src/autofill-utils.js

```diff
@@ -397,13 +402,30 @@ const wasAutofilledByChrome = (input) => {
 }
 
 /**
- * Checks if we should log debug info to the console
+ * Checks if we should log form analysis debug info to the console
  * @returns {boolean}
  */
 function shouldLog () {
+    return readDebugSetting('ddg-autofill-debug')
+}
```

Comment:
Just tiny abstractions on top of accessing `sessionStorage` so we don't have to duplicate the `try/catch` blocks.